app/src/main/java/de/westnordost/streetcomplete/quests/internet_access/AddInternetAccessForm.java

```diff
@@ -0,0 +1,55 @@
+package de.westnordost.streetcomplete.quests.internet_access;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddInternetAccessForm extends AbstractQuestAnswerFragment
+{
+	private static final String
+			YES = "yes",
```

Comment:
What's Yes?

Reply:
I mean, what would that be for example?

Reply:
The wiki says:
>  if there is public internet access, but the particular kind is unknown

But if we provide the option to select the particular kind of connection, this answer is IMO unnecessary.
I don't have a use case too...